openstack/octavia/ci/test-values.yaml

```diff
@@ -1,9 +1,11 @@
+---
 global:
   registry: myRegistry
   dbPassword: secret!
   master_password: topSecret!
   octavia_service_password: topSecret!!
-  dockerHubMirrorAlternateRegion: some_region
+  registryAlternateRegion: other.docker.registry
+  dockerHubMirrorAlternateRegion: other.dockerhub.mirror
```

Comment:
What are these needed for? Why is it okay to use dockerhub directly instead of keppel?

Reply:
We currently use dockerub mirror for upstream mariadb, memcached and rabbitmq images. The same is with database images like `percona/percona-xtradb-cluster:8.0.39-30.1` in `pxc-db` chart.